### ISSUE TITLE
Calling `list` on introspection module results in a java exception

### ISSUE THREAD

**Issue by ShreksHellraiser** [SwitchCraftCC/Plethora-Fabric#56]
On (as of writing) latest version of the SC3 pack.

CC:T 1.104.0

Plethora 1.5.6

Exception is

```
java.lang.NoSuchMethodError: 'java.util.Map dan200.computercraft.shared.peripheral.generic.methods.InventoryMethods.list(net.fabricmc.fabric.api.transfer.v1.item.InventoryStorage)'
	at io.sc3.plethora.integration.computercraft.InventoryMethodsWrapper.list(InventoryMethodsWrapper.kt:28) ~[Plethora-Fabric-1.5.6.jar:?]
	at io.sc3.plethora.api.method.BasicMethod$1.lambda$apply$0(BasicMethod.java:35) ~[Plethora-Fabric-1.5.6.jar:?]
	at io.sc3.plethora.core.executor.Task.update(Task.java:51) ~[Plethora-Fabric-1.5.6.jar:?]
	at io.sc3.plethora.core.executor.ComputerAccessExecutor$ComputerTask.update(ComputerAccessExecutor.java:163) ~[Plethora-Fabric-1.5.6.jar:?]
	at io.sc3.plethora.core.executor.TaskRunner.update(TaskRunner.java:32) ~[Plethora-Fabric-1.5.6.jar:?]
	at io.sc3.plethora.gameplay.manipulator.ManipulatorBlockEntity.tick(ManipulatorBlockEntity.java:250) ~[Plethora-Fabric-1.5.6.jar:?]
	at net.minecraft.class_2818$class_5563.method_31703(class_2818.java:665) ~[client-intermediary.jar:?]
	at net.minecraft.class_2818$class_5564.method_31703(class_2818.java:719) ~[client-intermediary.jar:?]
	at net.minecraft.class_1937.method_18471(class_1937.java:478) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_18765(class_3218.java:376) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:901) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:828) ~[client-intermediary.jar:?]
	at net.minecraft.class_1132.method_3748(class_1132.java:105) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:266) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

![image](https://user-images.githubusercontent.com/70228812/236953956-d40462bb-2175-4609-a4bf-7b1e721ab9bf.png)

**Comment by Kan18** [comment]
The CC:T version installed on the client is from GitHub, and includes API changes that Plethora has yet to adapt to. You can probably fix this by redownloading CC:T 1.104.0 from Modrinth

**Comment by ShreksHellraiser** [comment]
I've been informed this is due to the SC modpack client running ~~an older~~ a newer version of CC than the server.
